entries/o/onelogin.com.json

```
@@ -1,15 +1,23 @@
 {
   "OneLogin": {
     "domain": "onelogin.com",
-    "url": "https://www.onelogin.com/",
     "tfa": [
       "custom-software",
       "sms",
       "totp",
       "custom-hardware",
       "u2f"
     ],
-    "documentation": "https://onelogin.service-now.com/support?id=kb_article&sys_id=43d99143db109700d5505eea4b96197a",
+    "custom-software": [
+      "OneLogin Protect",
+      "Duo",
+      "RSA SecurID",
+      "Symantec VIP"
+    ],
+    "custom-hardware": [
+      "Yubico OTP"
+    ],
+    "documentation": "https://support.onelogin.com/kb/4266569",
```

```suggestion
    "documentation": "https://support.onelogin.com/kb/4266569/add-multi-factor-authentication",
```

The URL without the last bit kept throwing errors for me.